.github/workflows/build-and-test.yaml

```diff
@@ -20,3 +22,28 @@ jobs:
 
       - name: Build, Test
         run: docker compose up --build test
+
+  publish:
+    name: Publish image to Docker Hub
+    needs: build-test
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 #v3.5.3
+
+      - name: Create dummy .env
+        run: touch .env
```

Comment:
Minor, but I _think_ this step should be defunct if you're not building through `docker compose`